### ISSUE TITLE
Chromium and Firefox

### ISSUE THREAD

**Issue by sdanfa** [starfive-tech/Fedora_on_StarFive#43]
Hi,
Firefox and Chromium received riscv64 patches recently. Can you add Firefox and chromium in the packages for Fedora?

https://groups.google.com/a/groups.riscv.org/g/sw-dev/c/81caeTrQWLs

https://lists.riscv.org/g/software/topic/83352236

Thanks.

**Comment by tekkamanninja** [comment]
> Hi, Firefox and Chromium received riscv64 patches recently. Can you add Firefox and chromium in the packages for Fedora?
> 
> https://groups.google.com/a/groups.riscv.org/g/sw-dev/c/81caeTrQWLs
> 
> https://lists.riscv.org/g/software/topic/83352236
> 
> Thanks.

yes, we are already working on them ,  I think Firefox is nearly done, for Chromium, we are on debug and compilation step.

**Comment by sdanfa** [comment]
Okay thanks, please let me know once they are added to the repository. 👍😁
